ui-tests/cypress/support/pageObjects/app/Content.js

```diff
@@ -1,11 +1,11 @@
-import {htmlElements, WebElement} from '../WebElement.js';
+import {WebElement} from '../WebElement.js';
 
 export default class Content extends WebElement {
 
   getInputError(input) {
     return cy.get(input)
              .parent().parent()
-             .find(`${htmlElements.span}.help-block`);
+             .find(`.help-block`);
```

Comment:
Have you *double-checked* this is valid for all sections?

Reply:
I ran the tests where getInputError() recurred and found no problems